stable/xray/README.md

```diff
@@ -112,7 +112,7 @@ If Xray was installed with all of the default values (e.g. with no user-provided
 2. Upgrade the release by passing the previously auto-generated secrets:
 
 ```bash
-helm upgrade --name xray jfrog/xray --set rabbitmq-ha.rabbitmqPassword=<rabbit-password> --set postgresql.postgresqlPassword=<postgresql-password>
+helm upgrade --install xray -n xray jfrog/xray --set rabbitmq-ha.rabbitmqPassword=<rabbit-password> --set postgresql.postgresqlPassword=<postgresql-password>
```

Comment:
@shahiinn  Pls use —namespace instead of -n, so the same command works for both v2 and v3 helm